server/py/services/api/crud/model_monitoring/grafana.py

```diff
@@ -56,232 +52,57 @@ def grafana_list_projects(
     return projects_output.projects
 
 
-# The following functions were not removed due to backward compatibility that is related to iguazio version <= 3.5.2
-
-
-async def grafana_list_endpoints(
-    body: dict[str, Any],
+async def grafana_list_endpoints_uids(
     query_parameters: dict[str, str],
     auth_info: mlrun.common.schemas.AuthInfo,
-) -> list[mlrun.common.schemas.model_monitoring.grafana.GrafanaTable]:
-    project = query_parameters.get("project")
-
-    # Filters
-    model = query_parameters.get("model", None)
-    function = query_parameters.get("function", None)
-    labels = query_parameters.get("labels", "")
-    labels = labels.split(",") if labels else []
-
-    # Metrics to include
-    metrics = query_parameters.get("metrics", "")
-    metrics = metrics.split(",") if metrics else []
+    db_session: Session,
+) -> list[str]:
+    """
+    List available model endpoint uids. Will be used as a filter in each model endpoint grafana dashboard.
 
-    # Time range for metrics
-    start = body.get("rangeRaw", {}).get("start", "now-1h")
-    end = body.get("rangeRaw", {}).get("end", "now")
+    :param query_parameters:  Dictionary of query parameters attached to the request. Note that this parameter is
+                              required by the API even though it is not being used in this function.
+    :param auth_info:         The auth info of the request.
+    :param db_session:        A session that manages the current dialog with the database.
 
-    # Endpoint type filter - will be used to filter the router models
-    filter_router = query_parameters.get("filter_router", None)
+    :return: List model endpoints uids.
+    """
 
+    project = query_parameters.get("project")
     if project:
         await framework.utils.auth.verifier.AuthVerifier().query_project_permissions(
             project,
             mlrun.common.schemas.AuthorizationAction.read,
             auth_info,
         )
-    endpoint_list = await run_in_threadpool(
-        services.api.crud.ModelEndpoints().list_model_endpoints,
+    endpoint_list = await services.api.crud.ModelEndpoints().list_model_endpoints(
```

Comment:
You want to show all the meps or just the active?
If only the active (like the UI) you can use `latest_only=True`

Reply:
that's a good question, I think that for now as you suggested we should display only the active meps